.github/workflows/test-integrations-windows.yml

```diff
@@ -58,7 +58,7 @@ jobs:
     strategy:
       fail-fast: false
       matrix:
-        envoy-version: [ "1.24.10", "1.25.9", "1.26.4", "1.27.0" ]
+        envoy-version: [ "1.27.0" ]
```

Comment:
Are we maintaining compat with only the latest version or is this a temporary change?

Reply:
Please read description of PR.

Reply:
Not temporary.